### DISCUSSION TITLE
Location of log files

### DISCUSSION THREAD

**Discussion by tanishaagarwal346** [strimzi/strimzi-kafka-operator#9283] [Q&A] [answered]
I have deployed strimzi kafka on my kubernetes cluster. I want to know where the logs of kafka get stored in the pods in human readable form so that I can make them flow to splunk through fluentbit.

**Comment by scholzj** [comment]
The logs are collected through stdout / stderr by your container (e.g. containerd, CRI-O, etc.) engine and that is where Fluentbit should collect them.

**Reply by tanishaagarwal346** [reply]
So what is the location of the log in the pod?

**Reply by scholzj** [reply, marked as answer]
As I said, there is not log file anywhere int he Pod. It is on the worker node.
